datahub-web-react/src/graphql/glossaryTerm.graphql

```diff
@@ -42,5 +48,45 @@ query getGlossaryTerm($urn: String!, $start: Int, $count: Int) {
                 value
             }
         }
+         schemaMetadata(version: 0) {
+            ...schemaMetadataFields
+        }
+        previousSchemaMetadata: schemaMetadata(version: -1) {
+            ...schemaMetadataFields
+        }
+        editableSchemaMetadata {
+            editableSchemaFieldInfo {
+                fieldPath
+                description
+                globalTags {
+                    ...globalTagsFields
+                }
+            }
+        }
+
+    }
+}
+
+query getGlossaryTermSchemaVersions($urn: String!, $version1: Long, $version2: Long) {
+    glossaryTerm(urn: $urn) {
+        urn
+        name
+        type
+        hierarchicalName
+        schemaMetadata(version: $version1) {
+            ...schemaMetadataFields
+        }
+        previousSchemaMetadata: schemaMetadata(version: $version2) {
+            ...schemaMetadataFields
+        }
+        editableSchemaMetadata {
+            editableSchemaFieldInfo {
+                fieldPath
+                description
+                globalTags {
+                    ...globalTagsFields
+                }
+            }
+        }
```

Comment:
is this necessary? we have turned off schema version history and still have some work to do before we re-enable it. Can we get away w/ not supporting this for now?

Reply:
removed